vizro-core/hatch.toml

```diff
@@ -64,7 +64,8 @@ dependencies = [
   "mkdocs-material",
   "mkdocs-git-revision-date-localized-plugin",
   "mkdocstrings[python]",
-  "linkchecker"
+  "linkchecker",
+  "mkdocs-pycafe @ git+https://github.com/py-cafe/mkdocs-pycafe.git"
```

Comment:
Reminder to change this to package once released

Reply:
https://github.com/McK-Internal/vizro-internal/issues/1204